### PR TITLE
store: correct comments mistake.

### PR DESCRIPTION
**Pull request by hueypark**

### CODE REVIEW

File: store/tikv/2pc_slow_test.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ package tikv
 
 import . "github.com/pingcap/check"
 
+// TestCommitMultipleRegions tests commit multiple regions
```
Comment (zimulala) [inline store/tikv/2pc_slow_test.go]: Add `.` at the end of a line.

File: store/tikv/gcworker/gc_worker.go [hunk 2]
```diff
@@ -729,7 +729,7 @@ type gcTaskWorker struct {
 	store      tikv.Storage
 	taskCh     chan *gcTask
 	wg         *sync.WaitGroup
-	// use atomic to read and set
+	// successRegions and failedRegions use atomic to read and set
```
Comment (zimulala) [inline store/tikv/gcworker/gc_worker.go]: Ditto.

File: store/mockstore/mocktikv/rpc.go [hunk 3]
```diff
@@ -101,15 +101,15 @@ type rpcHandler struct {
 	cluster   *Cluster
 	mvccStore MVCCStore
 
-	// store id for current request
+	// storeID stores id for current request
 	storeID uint64
-	// Used for handling normal request.
+	// startKey used for handling normal request.
```
Comment (zimulala) [inline store/mockstore/mocktikv/rpc.go]: s/`used for`/`is used for`?

File: store/tikv/2pc_slow_test.go (hunk 1 above)
Comment (hueypark) [reply store/tikv/2pc_slow_test.go]: Thank you for your review.
Done.

File: store/tikv/gcworker/gc_worker.go (hunk 2 above)
Comment (hueypark) [reply store/tikv/gcworker/gc_worker.go]: Done.

File: store/mockstore/mocktikv/rpc.go (hunk 3 above)
Comment (hueypark) [reply store/mockstore/mocktikv/rpc.go]: Done.

**Pull request merged**